build.sbt

```diff
@@ -287,7 +288,17 @@ lazy val alpine = (project in file("alpine")) settings (
     // https://mvnrepository.com/artifact/com.fasterxml.jackson.module/jackson-module-scala
     "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.11.2",
 
-    // "org.scalatest" %% "scalatest" % "3.0.5" % "test",
-    "io.delta" %% "delta-core" % deltaVersion excludeAll (ExclusionRule("org.apache.hadoop"))
+    // Adding test classifier seems to break transitive resolution of the core dependencies
+    // i.e. needed for tests to compile
+    "org.apache.spark" %% "spark-core" % sparkVersion,
```

Comment:
Can we add the test scope and add missing dependencies? You can find the versions of these dependencies in this file: https://github.com/apache/spark/blob/v3.0.1/pom.xml

```
    "org.apache.spark" %% "spark-core" % sparkVersion % "test",
    "org.apache.spark" %% "spark-sql" % sparkVersion % "test",
```

Let's see what dependencies we really need.